scripts/setup-venv.sh

```diff
@@ -13,7 +13,7 @@ echo "python version: $(python --version)"
 if [[ "$OSTYPE" == "darwin"* ]]; then
     export HOMEBREW_NO_INSTALL_CLEANUP=1
     brew ls --versions gmp || brew install gmp
-    CFLAGS=-I`brew --prefix gmp`/include LDFLAGS=-L`brew --prefix gmp`/lib pip3 install fastecdsa
+    CFLAGS=-I`brew --prefix gmp`/include LDFLAGS=-L`brew --prefix gmp`/lib pip3 install fastecdsa  | { grep --invert-match 'Requirement already satisfied:' || :; }
```

Comment:
Same as https://github.com/Shard-Labs/starknet-hardhat-plugin/pull/296/files#r1115318562
Strips `Requirement already satisfied` logs and stops grep from exiting the process.